GradientColorPickerTest/IntegrationTest.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Drawing;
+using Messerli.GradientColorPicker;
+using Xunit;
+
+namespace Messerli.GradientColorPickerTest
+{
+    public class IntegrationTest
+    {
+        private const double GoodPasswordEntropy = 40.0;
+        private const double BaseBrightness = 191.0;
+        private static readonly GradientColorProvider GradientColorProvider = GradientColorBuilder
+            .WithColor(new GradientColorByValue(Color.FromArgb(255, 191, 191), 0))
+            .Add(new GradientColorByValue(Color.FromArgb(255, 255, 191), 20))
+            .Add(new GradientColorByValue(Color.FromArgb(191, 255, 191), 40))
+            .Build();
+
+        [Theory]
+        [MemberData(nameof(GetEntropy))]
+        public void PasswordStrengthColor(double entropy)
+        {
+            var oldVariantColor = GetOldVariantColor(entropy);
+            var newVariantColor = GetNewVariantColor(Convert.ToInt32(entropy));
+
+            Assert.True(newVariantColor.R - oldVariantColor.R <= 1);
```

Comment:
Use Assert.Equals(double, double, int precision)

Reply:
Addressed in: https://github.com/messerli-informatik-ag/gradient-color-picker/pull/4/commits/df77c75ae0f59c229a6bd4a4657f2ff50155d0ab

Reply:
Addressed in: https://github.com/messerli-informatik-ag/gradient-color-picker/pull/4/commits/ac8b87f45391e24525fc46845dfd6e0451fcb9b4

Reply:
Addressed in: https://github.com/messerli-informatik-ag/gradient-color-picker/pull/4/commits/ea4613b695bdba288b569ae6d7d4335c28503e4f